config/config.go

```diff
@@ -67,4 +69,12 @@ func (c *Connector) ApplyDefaults() {
 	if c.Kafka.MetadataTTL == 0 {
 		c.Kafka.MetadataTTL = 60 * time.Second
 	}
+
+	if c.Kafka.ProducerMaxAttempts == 0 {
+		c.Kafka.ProducerMaxAttempts = 10
```

Comment:
ProducerMaxAttempts need to be 
```
math.MaxInt
```
Also we need to set ProducerBatchBytes
```
math.MaxInt
```

Reply:
@halilkocaoz I have a few comments that I just realized.
And we need to update the README.md according to these changes.

Reply:
> ProducerMaxAttempts need to be `math.MaxInt`

kafka.Writer.MaxAttempts default value is 10. also @mhmtszr mentioned it, should we use `math.MaxInt` for MaxAttempts?

> Also we need to set ProducerBatchBytes `math.MaxInt`

It's default value is 1048576.

Should we use default values or not? What do you think about it?

Reply:
by default, `ProducerMaxAttempts` need to be infinite. we can use `math.MaxInt` for that.
for `ProducerBatchBytes` if default value is 1048576 i am okey with that.
basically, we need to preserve old behaviors.

Reply:
`ProducerMaxAttempts` default value is now math.MaxInt. `ProducerBatchBytes` is configured `10485760` as default. But it was statically setting for using math.MaxInt not from `ApplyDefaults()` func.